WcaOnRails/spec/models/user_spec.rb

```diff
@@ -645,4 +645,18 @@
       expect(senior_delegate.editable_fields_of_user(user).to_a).to include(:delegate_status, :senior_delegate_id, :region)
     end
   end
+
+  describe "birthdate validations" do
+    it "requires birthdate in past" do
+      user = FactoryBot.create :user
```

Comment:
This appears twice and is a perfect candidate to make use of `let`, like so:
```ruby
describe "birthdate validations" do
  let(:user) { FactoryBot.create :user }

  it "requires birthdate in past" do
    user.dob = 5.days.from_now
    expect(user).to be_invalid_with_errors(dob: ["must be in the past"])
  end
  
   it "requires user over two years old" do
    user.dob = 5.days.ago
    expect(user).to be_invalid_with_errors(dob: ["must be at least two years old"])
  end
end
```
What [`let`](https://relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let) really gives us, is the ability to move such variable definition out of a single test and reuse it across many of those (while providing each test with a fresh value). Usually it removes repetition and perhaps make the tests itself a bit cleaner.
In cases like this it's somehow a matter of preference though, so I don't feel to strongly about this =)